components/ui/Feed/FeedSortButton.tsx

```diff
@@ -0,0 +1,62 @@
+import { useTheme } from "native-base";
```

Comment:
Same philosophy as the FeedOverflowMenuButton. No need to set state in the FeedView for the sort button, as it's all based on the `SortType` from the Lemmy api anyways. Now we can just pass it in, and the component can render whatever component it wants.